client/components/file-upload/index.tsx

```diff
@@ -15,12 +15,27 @@ import { BaseControl } from 'wcpay/components/wp-components-wrapped/components/b
 import { DropZone } from 'wcpay/components/wp-components-wrapped/components/drop-zone';
 import { FormFileUpload } from 'wcpay/components/wp-components-wrapped/components/form-file-upload';
 import { Button } from 'wcpay/components/wp-components-wrapped/components/button';
-import type { FileUploadControlProps } from 'wcpay/types/disputes';
 import FileUploadError from './upload-error';
 import FileUploadPreview from './preview';
 
+interface FileUploadControlProps {
+	fieldKey: string;
+	fileName: string;
+	disabled?: boolean;
+	isDone: boolean;
+	isLoading: boolean;
+	accept: string;
+	error?: string;
+	onFileChange( key: string, file: File ): Promise< void >;
+	onFileRemove( key: string ): void;
+	help?: string;
+	showPreview?: boolean;
+	uploadButtonLabel?: string;
+	type?: string;
+}
+
 export const FileUploadControl = ( {
-	field,
```

Comment:
The `field` object had a `key` and `name` property. But the `name` property wasn't used anywhere.